src/renderer/components/+cluster-settings/components/install-feature.tsx

```diff
@@ -1,44 +1,48 @@
 import React from "react";
 import { observable, reaction, comparer } from "mobx";
 import { observer, disposeOnUnmount } from "mobx-react";
-import { clusterIpc } from "../../../../common/cluster-ipc";
 import { Cluster } from "../../../../main/cluster";
 import { Button } from "../../button";
 import { Notifications } from "../../notifications";
 import { Spinner } from "../../spinner";
+import { Feature, FeatureStatus } from "../../../../main/feature";
 
 interface Props {
   cluster: Cluster
-  feature: string
+  feature: Feature
 }
 
 @observer
 export class InstallFeature extends React.Component<Props> {
   @observable loading = false;
+  @observable status: FeatureStatus
 
   componentDidMount() {
+    this.props.feature.featureStatus(this.props.cluster).then((status) => {
+      this.status = status
+    })
     disposeOnUnmount(this,
-      reaction(() => this.props.cluster.features[this.props.feature], () => {
+      reaction(() => this.status, () => {
         this.loading = false;
       }, { equals: comparer.structural })
     );
   }
 
   getActionButtons() {
     const { cluster, feature } = this.props;
-    const features = cluster.features[feature];
     const disabled = !cluster.isAdmin || this.loading;
     const loadingIcon = this.loading ? <Spinner/> : null;
```

Comment:
The spinner never stops after initiating install (or uninstall). `this.status` is not getting updated? I can't see where the problem is...

Reply:
Will fix this.